src/platform/plugins/shared/discover/public/application/main/components/top_nav/save_discover_session/save_modal.tsx

```diff
@@ -68,22 +81,15 @@ export const DiscoverSessionSaveModal: React.FC<DiscoverSessionSaveModalProps> =
   ) : null;
 
   const timeSwitch = isTimeBased ? (
-    <EuiFormRow
-      helpText={
-        <FormattedMessage
-          id="discover.topNav.saveModal.storeTimeWithSearchToggleDescription"
-          defaultMessage="Update the time filter and refresh interval to the current selection when using this session."
-        />
-      }
-    >
+    <EuiFormRow fullWidth>
       <EuiSwitch
         data-test-subj="storeTimeWithSearch"
         checked={timeRestore}
         onChange={(event) => setTimeRestore(event.target.checked)}
         label={
           <FormattedMessage
             id="discover.topNav.saveModal.storeTimeWithSearchToggleLabel"
-            defaultMessage="Store time with Discover session"
+            defaultMessage="Include time range and refresh interval customizations"
```

Comment:
It reads like 2 verbs with `and` in between: `include` and `refresh` which makes it confusing

Reply:
```suggestion
            defaultMessage="Include time range customizations"
```